src/main/java/ca/concordia/soen6441/project/gameplay/orders/Airlift.java

```diff
@@ -1,11 +1,95 @@
 package ca.concordia.soen6441.project.gameplay.orders;
 
-import ca.concordia.soen6441.project.interfaces.Order;
+import ca.concordia.soen6441.project.interfaces.Country;
+import ca.concordia.soen6441.project.interfaces.Player;
+import ca.concordia.soen6441.project.interfaces.context.HandOfCardsContext;
+import ca.concordia.soen6441.project.log.LogEntryBuffer;
+import ca.concordia.soen6441.project.gameplay.cards.AirliftCard;
+import ca.concordia.soen6441.project.context.hand.CardManager;
+import ca.concordia.soen6441.project.interfaces.context.GameContext;
 
-public class Airlift implements Order {
+/**
+ * The Airlift order allows moving troops from one country to another,
+ * even if they are not adjacent. It extends Advance but removes neighbor restrictions.
+ */
+public class Airlift extends Advance {

```

Comment:
Airlift shouldnt extend Advance, given the requirements since it doesnt attack.  Please change this ASAP.

Also, you need to implement your toString method.  Look at whats done for Advance, and do something similar for AIrlift.